inc/compat/plugins/compat-plugin-mailpoet.php

```diff
@@ -0,0 +1,135 @@
+<?php
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Compatibility with plugin: MailPoet integration.
+ */
+class FluidCheckout_MailPoet extends FluidCheckout {
+
+    /**
+     * __construct function.
+     */
+    public function __construct( ) {
+        add_action( 'init', [ $this, 'hooks' ] );
```

Comment:
@alamgircsebd please always use `array()` to instantiate an array instead of `[]` for better legibility and to match the rest of the code style.

It is only acceptable to use `[]` when copying code from other plugins that are necessary, such as private functions from the other plugins that we need to use or modify.

**Please go over the entire class and replace the initialization of arrays with `array()`.**